modules/custom/d_p/js/d_p_ckeditor.js

```diff
@@ -7,33 +7,46 @@
-          link.href = '/profiles/contrib/droopler/modules/custom/d_p/css/d_p_ckeditor.css';
-          head.appendChild(link);
-        }
-      }
-
+      // @todo: Rework this to better solution when it will be possible.
       function d_p_ckeditor() {
-        if (typeof CKEDITOR !== "undefined") {
-          for (let instanceName in CKEDITOR.instances) {
-            if (CKEDITOR.instances[instanceName].element.$.parentElement.classList.contains('d_p_ckeditor_centered')) {
-              if (typeof CKEDITOR.instances[instanceName].document !== 'undefined') {
-                d_p_ckeditor_add_js(CKEDITOR.instances[instanceName].document.$);
-              } else {
-                setTimeout(function () {
-                  d_p_ckeditor_add_js(CKEDITOR.instances[instanceName].document.$);
-                }, 500);
+        if (typeof Drupal.CKEditor5Instances !== "undefined") {
+          $(once('d_p_ckeditor_centered', '.d_p_ckeditor_centered', context)).each(function () {
+            let editor_id = $(this).find('textarea').attr('data-ckeditor5-id');
+            let editor = Drupal.CKEditor5Instances.get(editor_id);
+            let $editor_editable = $(this).find('.ck-editor__editable');
+
+            // Set default alignment to center.
+            $editor_editable.children().each(function () {
+              if (!$(this).hasClass('text-align-left') && !$(this).hasClass('text-align-right') && !$(this).hasClass('.text-align-justify')) {
+                $(this).addClass('text-align-center');
               }
-            }
-          }
+            });
+            editor.setData($editor_editable.html());
+
+            // Add text-align-left class to selected element.
+            $(once('open_alignment_dropdown', '.ck.ck-dropdown.ck-toolbar-dropdown.ck-alignment-dropdown')).on('click', function() {
+              $(once('alignment_select', '.ck.ck-dropdown.ck-toolbar-dropdown.ck-alignment-dropdown .ck.ck-toolbar__items button')).on('click', function() {
+                let current_alignment = editor.commands.get('alignment').value;
+                if (current_alignment === 'left') {
+                  editor.editing.view.change(writer => {
+                    if (typeof editor?.editing?.view?.document?.selection?.focus?.parent?.parent !== 'undefined') {
+                      writer.addClass('text-align-left', editor.editing.view.document.selection.focus.parent.parent);
+                    }
+                  });
+                }
+                else {
+                  editor.editing.view.change(writer => {
+                    if (typeof editor?.editing?.view?.document?.selection?.focus?.parent?.parent !== 'undefined') {
+                      writer.removeClass('text-align-left', editor.editing.view.document.selection.focus.parent.parent);
+                    }
+                  });
+                }
+                setTimeout(function() {
+                  editor.setData($editor_editable.html());
```

Comment:
is there some better way to commit the changes in the writter? Without this line, the changes are visible, but they are not stored after save.

Reply:
Changes need to be made in the model. Applying them on the view layer, without changing the model is a bit like hot fixing a website with some JS without actually changing the underlying database.   
  
Maybe another important thing to understand here is that you're patching here only the editing view. While the data view (which is a separate one, created on the fly when getData() is called) is not affected.

Reply:
Changes need to be made in the model. Applying them on the view layer, without changing the model is a bit like hot fixing a website with some JS without actually changing the underlying database.   
  
Maybe another important thing to understand here is that you're patching here only the editing view. While the data view (which is a separate one, created on the fly when getData() is called) is not affected.